### PR TITLE
Use named import for `fast-json-patch` update to 2.2.0

### LINKED ISSUE #1460: Swagger Client Initialisation is failing.
Getting a sudden error while initialisation of the Swagger.

```
[ TypeError: Cannot read property 'applyPatch' of undefined
      at Object.a [as applyPatch] (/path/to/node_modules/swagger-client/dist/index.js:1:9824)
      at e.value (/path/to/node_modules/swagger-client/dist/index.js:1:24637)
      at /path/to/node_modules/swagger-client/dist/index.js:1:24394
      at Array.forEach (<anonymous>)
      at e.value (/path/to/node_modules/swagger-client/dist/index.js:1:23989)
      at new e (/path/to/node_modules/swagger-client/dist/index.js:1:20400)
      at a (/path/to/node_modules/swagger-client/dist/index.js:1:19328)
      at t (/path/to/node_modules/swagger-client/dist/index.js:1:14280)
      at Function.o [as resolve] (/path/to/node_modules/swagger-client/dist/index.js:1:14995)
      at a.resolve (/path/to/node_modules/swagger-client/dist/index.js:1:18398)
      at new a (/path/to/node_modules/swagger-client/dist/index.js:1:17450)
      at a (/path/to/node_modules/swagger-client/dist/index.js:1:17408)
      at Promise (/path/to/node_modules/connector-compliance-tests/classes/swaggerApiHelper.js:188:7)
      at new WrappedPromise (/path/to/node_modules/async-listener/es6-wrapped-promise.js:13:18)
      at ApiHelper.doClientInitialisation (/path/to/node_modules/connector-compliance-tests/classes/swaggerApiHelper.js:187:12)
      at Promise.all.swaggerObjectList.map (/path/to/node_modules/connector-compliance-tests/validators/swaggerValidator.js:90:31)
      at Array.map (<anonymous>)
      at Promise (/path/to/node_modules/connector-compliance-tests/validators/swaggerValidator.js:88:35)
      at new WrappedPromise (/path/to/node_modules/async-listener/es6-wrapped-promise.js:13:18)
      at Object.setupSwaggerClient (/path/to/node_modules/connector-compliance-tests/validators/swaggerValidator.js:86:10)
      at Context.<anonymous> (/path/to/node_modules/connector-compliance-tests/testSuite/discoverySwaggerSchemaTests.js:43:32)
      at callFnAsync (/path/to/node_modules/mocha/lib/runnable.js:371:21)
      at Test.Runnable.run (/path/to/node_modules/mocha/lib/runnable.js:318:7)
      at Runner.runTest (/path/to/node_modules/mocha/lib/runner.js:443:10)
      at /path/to/node_modules/mocha/lib/runner.js:549:12
      at next (/path/to/node_modules/mocha/lib/runner.js:361:14)
      at /path/to/node_modules/mocha/lib/runner.js:371:7
      at next (/path/to/node_modules/mocha/lib/runner.js:295:14)
      at Immediate._onImmediate (/path/to/node_modules/mocha/lib/runner.js:339:5)
      at processImmediate (internal/timers.js:443:21) ]

```

Swagger Client version 3.9.0

### PR DESCRIPTION
**Pull request by tomalec**
### Description

- Changed default import to named import for `fast-json-patch`
- Updated `fast-json-patch` back to the current `2.2.0`

### Motivation and Context

As reported at https://github.com/Starcounter-Jack/JSON-Patch/issues/233 and https://github.com/swagger-api/swagger-js/issues/1460 (Fixes #1460) there was a problem initializing Swagger with latest `fast-json-patch`. It occurred to be related to backward incompatible change related to undocumented default export. We will revert this change, and release a new version to support this and many other projects that are currently using `import jsonPatch from 'fast-json-patch'`.

However, we recommend to use named or star imports:
```js
import { applyPatch } from 'fast-json-patch'
import * as jsonPatch from `fast-json-patch'
``` 
Those worked with old versions, works fine with 2.2.0, and we plan long term support for such import.
So you can use it with current versions without a need to wait for our fix.

### Types of changes

- [ ] No code changes (changes to documentation, CI, metadata, etc)
- [x] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] My code follows the code style of this project. - at least I hope so, I haven't found any guidelines, nor CONTRIBUTING.md
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed. - I tried running `npm test` locally, but they were failing even on master :(

### CODE REVIEW

File: src/specmap/lib/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import jsonPatch from 'fast-json-patch'
+import { applyPatch as fastApplyPatch, getValueByPointer} from 'fast-json-patch'
```
Comment (tim-lai) [inline src/specmap/lib/index.js:1]:
```suggestion
import {applyPatch as fastApplyPatch, getValueByPointer} from 'fast-json-patch'
```

File: package.json [hunk 2]
```diff
@@ -104,7 +104,7 @@
     "cross-fetch": "0.0.8",
     "deep-extend": "^0.5.1",
     "encode-3986": "^1.0.0",
-    "fast-json-patch": "~2.1.0",
+    "fast-json-patch": "^2.2.0",
```
Comment (tim-lai) [inline package.json:107]:
```suggestion
    "fast-json-patch": "^2.2.1",
```

**Comment by tomalec** [comment]
@tim-lai thanks for the review.
All comments addressed.

**Comment by tim-lai** [comment]
@tomalec please resolve merge conflicts as well. Thanks!

**Comment by char0n** [comment]
@tomalec thank you for reporting this! Closing this PR in favor of https://github.com/swagger-api/swagger-js/pull/1564

**Pull request closed without merging**